src/views/bibliography_item.tsx

```diff
@@ -79,6 +79,22 @@ export class BibliographyItemView<PropsType extends BibliographyItemViewProps>
       components: { ReferencesEditor },
     } = this.props
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const handleSave = async (data: any) => {
```

Comment:
that should be Partial< Model > I think instead of any

Reply:
Updated to be `Partial<BibliographyItem>`.